src/useOutsideClick.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+export function useOutsideClick(
+   ref: React.RefObject<HTMLElement>,
+   handler: (event: Event) => void
+) {
+   React.useEffect(() => {
+      const listener = (event: Event) => {
+         const target = event.target as HTMLElement;
+         if (ref.current?.contains(target) === false) {
```

Comment:
```suggestion
         if (!ref.current?.contains(target)) {
```

Reply:
I think this is specifically trying to check that ref.current exists and does not contain the target?

Reply:
Ah I see.